frontstage/templates/surveys/surveys-share/send-instructions.html

```diff
@@ -76,12 +52,15 @@
             {% endfor %}
         {% endif %}
     {% endwith %}
-    <h1 class="ons-u-fs-xl">Send instructions</h1>
-    <p>We will send an email to <strong>{{ email }}</strong> with instructions to access the following surveys:</p>
+    <div id="main_page_text">
+        <h1 class="ons-u-mb-s">Send instructions</h1>
+        <p>We will email a link with instructions to <strong>{{ email }}</strong>.</p>
+        <p class="ons-u-mb-s">Once approved, they will have access to: </p>
+    </div>
     {% for business in share_dict %}
-        <h2 class="ons-u-fs-m">{{ share_dict[business].name }}</h2>
+        <h2>{{ share_dict[business].name }}</h2>
```

Comment:
The docs show this as `Business:` though one of the assumptions is that business is changed to organisation on all pages so think this needs to be 

`Organisation: {{ share_dict[business].name }}`